site/_includes/partials/meta.njk

```diff
@@ -26,7 +26,7 @@
 <meta property="og:type" content="website" />
 <meta property="og:url" content="{{ currentUrl }}" />
 
-{% set socialImage = thumbnail or hero or site.defaultSocialImage %}
+{% set socialImage = sharingimage or thumbnail or hero or site.defaultSocialImage %}
```

Comment:
Please use camelcase or underscore in var names.

Reply:
I updated this